ansible/vsonic_vtb

```diff
@@ -0,0 +1,228 @@
+all:
+  children:
+    vm_host:
+      children:
+        vm_host_1:
+    vsonic:
```

Comment:
@lguohan 
> why are we duplicating this veos_vtb file to vsonic_vtb? it looks the same as veos_vtb.

Yes, I almost duplicate the file `veos_vtb`, but I change the group name of `eos` to `vsonic` at here. Is it not proper if I want to change the inventory of Ansible ?

Reply:
i'd like to remove the redudency here. can we still use veos_vtb and put vms_1 as children of both vsonic and veos?

Reply:
Remove vsonic_vtb. Please review it.